docs/data/date-pickers/custom-components/custom-components.md

```diff
@@ -80,3 +80,14 @@ This can be used in combination with `componentsProps`.
 In the example below, the actions are the same as in the section above, but they are rendered inside a menu:
 
 {{"demo": "ActionBarComponent.js"}}
+
+## Paper content
+
+The paper content is available on all desktop picker components.  
+It adds a flexible way to extend what is rendered in the picker paper.  
+You can provide any custom component to this slot as long as it passes `children` down maintaining original picker behavior.  
+In the examples below we provide ways of implementing date range shortcuts using this slot.
+
+{{"demo": "PaperContentComponent.js"}}
+
+{{"demo": "StaticPaperContentComponent.js"}}
```

Comment:
Should the two demo be merged in a single one?

Reply:
How would you image them being merged?
Render both components in a single `demo`? 🤔

Reply:
Yes, in other demo when we display the mobile, desktop, and static components